src/reference/data-formats.rst

```diff
@@ -30,6 +30,34 @@ When using `tsv-utils <https://opensource.ebay.com/tsv-utils/>`__
     | tsv-uniq -H -f strain \
     | csvtk fix-quotes --tabs > output.tsv
 
+If you are using custom Python scripts to handle TSV files, we recommend using the
+`csv module <https://docs.python.org/3/library/csv.html>`__ to read and write the files.
+
+.. note::
+
+  Be sure to follow `csv module's recommendation <https://docs.python.org/3/library/csv.html#id4>`__
+  to open files with the ``newline=''``.
+
+Reading a TSV file:
+
+.. code-block:: Python
+
+  with open(input_file, 'r', newline='') as handle:
+    reader = csv.reader(handle, delimiter='\t')
+    for row in reader:
+      ...
+
+Writing a TSV file:
+
+.. code-block:: Python
+
+  with open_file(output_file, 'w', newline='') as output_handle:
```

Comment:
Consistently use `open` or `open_file`. I think `open` is better. If using `open_file`, an import statement should be added.

```suggestion
  with open(output_file, 'w', newline='') as output_handle:
```

Reply:
Ah, thanks for catching that! It was definitely a copy/paste error.